pages/explorer/races/[raceId]/units/[unitId].tsx

```diff
@@ -424,6 +424,19 @@ const createdCalculateValuesForUnits = (
     }
   }
 
+  // Some abilities are duplicated, they have different IDs but the name and description is the same.
+  // IF the UI is completely the same, we can remove the duplicates.
+  // This might lead to some bugs in case other parts would be different.
+  const abilities: AbilitiesType[] = [];
+  for (const ability of rawAbilities) {
+    if (!abilities.find((x) => JSON.stringify(x.ui) === JSON.stringify(ability.ui))) {
+      // If we are missing the name of the ability --> it's most likely broken // remove it here so we save the data
+      if (ability.ui.screenName) {
+        abilities.push(ability);
+      }
+    }
+  }
```

Comment:
@KingDarBoja  some unites have their abilities duplicated / I tried to remove it when the UI is the same. 
I hope this won't cause problems.

![image](https://github.com/user-attachments/assets/036a796d-35e5-4f5e-aac8-a16762f81847)
![image](https://github.com/user-attachments/assets/05b86032-0ce4-4b54-aa33-0f42cc06a31a)